index.html

```diff
@@ -327,7 +328,7 @@ <h1>Get Started</h1>
         </div>
 
         <div id="get-started" class="form">
-            <form action=".">
+            <form action="https://formspree.io/f/myyldona" method="POST">
                 <div class="formcontainer">
                     <div class="fistname">
                         <label for="fname">First Name</label>
```

Comment:
- [ ] The `name` input should have a `max length` attribute and `required` attribute to activate the validation on the form submit. Kindly fix it.

Reply:
Changed. Thanks for this review.